src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,51 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post,
+}) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <>
+      {user
+      && (
```

Comment:
I guess if you don't have a user then you can render everything else except "posted by" part
```suggestion
```

Reply:
moved to dedicated xml point